### PR TITLE
Fix material-ui error on WrapperWidgetUI

### PR DESCRIPTION
**Pull request by builtbyedgar**

### CODE REVIEW

**Comment by VictorVelarde** [comment]
Looks good. Do you have any reference to the original error?
It will be also nice to add a line to the CHANGELOG

**Comment by neokore** [comment]
The error message is this:
image

File: src/ui/widgets/WrapperWidgetUI.js [hunk 1]
```diff
@@ -162,22 +162,23 @@ function WrapperWidgetUI(props) {
...
+                getContentAnchorEl={null}
                 keepMounted
                 open={open}
                 onClose={handleClose}
                 PaperProps={{
                   style: {
                     marginTop: '48px',
-                    maxHeight: '144px',
-                    width: '128px',
+                    maxHeight: '168px',
```
Comment (neokore) [inline src/ui/widgets/WrapperWidgetUI.js:180]: Migrate it to `theme.spacing` using `useTheme` hook

**Comment by neokore** [comment]
Missing a changelog entry only, for the rest, it is OK

File: src/ui/widgets/WrapperWidgetUI.js (hunk 1 above)
Comment (VictorVelarde) [reply src/ui/widgets/WrapperWidgetUI.js]: By defeault 1 spacing = 8, so 
```
marginTop:  theme.spacing(8),
maxHeight:  theme.spacing(21),
```

**Comment by VictorVelarde** [comment]
@neokore why develop and not master?

**Comment by neokore** [comment]
That's because this PR is old. @builtbyedgar already opened another PR (https://github.com/CartoDB/carto-react-lib/pull/55) because this one was hard to rebase. I'll close it.

**Pull request closed without merging**